docs/_static/css/custom.css

```diff
@@ -0,0 +1,21 @@
+html[data-theme="light"] {
+    /* match pst-color-text-muted (used in header buttons) */
+    --scipp-filter-header-icon: saturate(0) brightness(0.696);
+}
+
+html[data-theme="dark"] {
+    /* match pst-color-text-muted (used in header buttons) */
+    --scipp-filter-header-icon: saturate(0) brightness(1.161);
+}
```

Comment:
Not sure I get this, are you changing the logo color, or the header colors?

Reply:
The logo colour to match the colour of the other icons and text items in the header.

Reply:
I am not sure we want to change the logo color (not necessarily saying we don't)? Shouldn't we rather match the style to the logo?

Reply:
What do you mean? Do you want to make all other icons anaconda-green as well?

Reply:
> It is a little bit of a hack to make the logo match the colours of the other header items. 

So when you say "logo" you do not mean our logo, but the anaconda icon?

Reply:
Yes.
Have a look at the current Plopp docs. They have a green anaconda logo in the top-right. The current Sciline docs don't have a logo but the text 'conda'. With this PR, both will end up with a conda logo that is styled like the PyPI, GH, etc icons.

Reply:
👍